plugins/woocommerce/tests/e2e-pw/tests/merchant/create-variable-product.spec.js

```diff
@@ -52,23 +52,19 @@ test.describe.serial( 'Add New Variable Product Page', () => {
 
 		// add 3 attributes
 		for ( let i = 0; i < 3; i++ ) {
-			if ( i > 0 ) {
-				await page.click( 'button.add_attribute' );
-			}
```

Comment:
I think we still need this if-condition because starting at WC 7.5 (I think, or 7.6?) an empty "Add custom attribute" form will already be shown when the user navigates to the "Attributes" tab.

Without this if-condition, the "Attributes" tab will contain 1 empty attribute form before hitting "Save attributes" like shown here:
<img width="495" alt="q8AUlMl5jE" src="https://user-images.githubusercontent.com/4509348/226844315-b2a779cc-9163-4994-a712-46ea5c56c6ea.png">

Reply:
I think my local must have had some issues.  I'm seeing that now, but I wasn't at first.